README.md

```diff
@@ -18,6 +18,23 @@ steps:
 - run: icoFoam
 ```
 
+### testing mulitple OpenFOAM versions
+
+```yaml
+strategy:
+  matrix:
+    openfoam-version: [2312, 2406, 2412]  # Add other versions here if needed
+
```

Comment:
```suggestion
  fail-fast: true
```

IME most of the times one actually wants this; might as well suggest it here. What do you think?